scss/utilities/_borders.scss

```diff
@@ -54,6 +54,10 @@
   border-radius: 50% !important;
 }
 
+.rounded-pill {
+  border-radius: 50px !important;
```

Comment:
I get the same result with 20px. Can't we use a relative unit here?

Reply:
I've always used `50px`, the problem you'll absolutely find with using `20px` is it works fine with small buttons, but larger buttons wouldn't work.

I'd recommend going with `50px` as it will guarantee to work regardless of button size. If we're to use `em` rather than `px` then it would need to be: `3.125em`

Reply:
If you could make that adjustment that would be great, it's also a mixin too.